src/components/Modals/EditExpertProfileModal.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useContext, useRef, useState } from 'react';
+import DialogModalBase from './DialogModalBase';
+import styles from './EditExpertProfileModal.module.scss';
+import Context from '../../context';
+import { ModalContext } from '../../context/ModalContext';
+
+const accentColor = '#4E6AE6';
+
+const EditExpertProfileModal = (props) => {
+  const [email, setEmail] = useState(null);
+  const [expertise, setExpertise] = useState(null);
+  const [description, setDescription] = useState(null);
+  const [expertiseTags, setExpertiseTags] = useState(null);
+  const [visibility, setVisibility] = useState(null);
+
+  const [fieldsMissing, setFieldsMissing] = useState(false);
+
+  const apiContext = useContext(Context.Api);
+  const { user } = useContext(Context.User);
+  const modalContext = useContext(ModalContext);
+
```

Comment:
It would be nice if the fields would be initialized with the user's existing expert data. That is accessible as `user.expertData`.

Reply:
Fixed in 3a687e1458e47e8e60698eadfb36d91abd70d3f1